src/pages/Profile/index.js

```diff
@@ -38,13 +39,13 @@ class Profile extends React.Component {
       position: React.createRef(),
     };
     this.state = {
-      dataExpireValue: '6 months',
-      editView: false,
-      displayName: 'Anisha Swain',
-      emailVal: 'xyz.redhat@gmail.com',
-      positionVal: 'Associate Software Engineer',
-      accountType: 'Pbench',
-      accountStatus: 'Active',
+      dataExpireValue: profileData.dataExpireOptions,
+      editView: profileData.editView,
+      displayName: profileData.displayName,
+      emailVal: profileData.emailVal,
+      positionVal: profileData.positionVal,
```

Comment:
And why wouldn't companyID be in the state?

Reply:
added `companyID` to state